sunpy/map/sources/adapt.py

```diff
@@ -0,0 +1,37 @@
+"""
+ADAPT Map subclass definitions
+"""
+
+from astropy.time import Time
+
+from sunpy.map.mapbase import GenericMap, SpatialPair
+
+__all__ = ['ADAPTMap']
+
+class ADAPTMap(GenericMap):
+    """
+    ADAPT (Air Force Data Assimilative Photospheric Flux Transport) flux transport model evolves an ensemble of realizations.
+
+    References
+    ----------
+    * `NSO ADAPT Page <https://nso.edu/data/nisp-data/adapt-maps/>`__
+    * `Data archive <https://gong.nso.edu/adapt/maps/>`__
+    """
+    @property
+    def coordinate_system(self):
+        ctype1, ctype2 = self.meta['ctype1'], self.meta['ctype2']
+        if ctype1 == 'Long':
+            ctype1 = 'CRLN-CAR'
+        if ctype2 == 'Lat':
+            ctype2 = 'CRLT-CAR'
+        return SpatialPair(ctype1, ctype2)
+
+    @property
+    def date(self):
+        return Time(self.meta.get('date-obs',self.meta.get('maptime')))
```

Comment:
This should probably fall back to `super().date` if neither of those keys can be found.

Reply:
I can't python, please hold

Reply:
Python better?